proto/const.go

```diff
@@ -21,24 +21,28 @@ const (
 	STATE_FAIL    // failed due to error or non-zero exit
 	STATE_TIMEOUT // timeout
 	STATE_STOPPED // stopped by user
+
+	STATE_SUSPENDED // suspended; doesn't apply to jobs
```

Comment:
What does "doesn't apply to jobs" mean? The other states do apply but this one doesn't?